### PR TITLE
LLP-17-CreditPage

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/pages/credits.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import styles from "@/styles/credits.module.scss"
+import IndexLayout from "@/Layouts/IndexLayout"
+import Accordion from "@/components/Accordion"
+import Link from "next/link"
+
+
```
Comment (pinenetcoder) [inline src/pages/credits.js:6]: Delete extra empty spaces please.

File: src/pages/payments.js [hunk 2]
```diff
@@ -20,82 +20,81 @@ export default function Payments() {
...
+                                 <li>Nespauskite jokių el. paštu, pokalbių programėlėse ar SMS žinutėse gautų nuorodų,
+                                    nevykdykite prašymų suvesti arba padiktuoti prisijungimo prie interneto banko duomenis.
+                                 </li>
+                                 <li>Atidžiai perskaitykite gautos internetinės nuorodos adresą – sukčiai dažnai sukuria beveik
+                                    identišką interneto banko prisijungimo puslapį, bet jo nuoroda nuo tikrojo visada skirsis bent
+                                    viena raide ar simboliu.
+                                 </li>
+                              </ul>
+
+
```
Comment (pinenetcoder) [inline src/pages/payments.js:91]: Delete extra empty spaces, please.

File: src/pages/credits.js (hunk 1 above)
Comment (mdainys) [reply src/pages/credits.js:6]: Done

File: src/pages/payments.js (hunk 2 above)
Comment (mdainys) [reply src/pages/payments.js:91]: Done

File: src/styles/credits.module.scss [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      border-right: solid 1px var(--theme-color4);
+      background-color: var(--main-background);
+      transform:
+         rotate(45deg) translate(399px, -241px);
+   }
+}
+
+
+
+
```
[the remark this answers is not in the document]
Comment (mdainys) [reply src/styles/credits.module.scss:187]: Done

**Pull request merged**